packages/api-contract/src/RxBlueprint.ts

```diff
@@ -46,7 +46,7 @@ export default class Blueprint extends RxBase {
   public deployContract (endowment: number | BN, maxGas: number | BN, ...params: any[]): BlueprintCreate {
     const signAndSend = (account: IKeyringPair | string | AccountId | Address): BlueprintCreateResultSubscription => {
       return this.apiContracts
-        .create(endowment, maxGas, this.codeHash, this.abi.deploy(...params))
+        .create(endowment, maxGas, this.codeHash, this.abi.constructors[0](...params))
```

Comment:
We should really add a `TODO` here - we probably want to specify exactly the constructor to be used in the longer-term, not just default to the first.

Reply:
Good call, put something in there I think is right but lmk